src/main/java/htsjdk/samtools/filter/TagFilter.java

```diff
@@ -60,6 +61,32 @@ public TagFilter(String tag, List<Object> values) {
         this.values = values;
     }
 
+    /**
+     * Constructor for a single value
+     *
+     * @param tag           the key of the tag to match
+     * @param value         the value to match
+     * @param includeReads  whether to include or not include reads that match filter
+     */
+    public TagFilter(String tag, Object value, final boolean includeReads) {

```

Comment:
could you modify the old constructors to delegate to the new constructors instead of duplicating code?

infact, I'd only keep around the actual code for TagFilter(String tag, List<Object> values, final boolean includeReads) and everything else should delegate via `this(...,..,...)`